packages/sources/tradingeconomics-test/src/endpoint/price-router.ts

```diff
@@ -0,0 +1,103 @@
+import { RoutingTransport } from '@chainlink/external-adapter-framework/transports/meta'
+import { wsTransport } from './price-ws'
+import { customSettings } from '../config'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { batchTransport } from './price'
+import {
+  PriceEndpoint,
+  PriceEndpointInputParameters,
+} from '@chainlink/external-adapter-framework/adapter'
+import { InputParameters } from '@chainlink/external-adapter-framework/validation/input-params'
+
+export const inputParameters: InputParameters & PriceEndpointInputParameters = {
```

Comment:
I think you can define these params as just `PriceEndpointInputParameters`
```suggestion
export const inputParameters: PriceEndpointInputParameters = {
```